docs/eog/case-studies-cw20.rst

```diff
@@ -0,0 +1,44 @@
+.. _CW20:
+
+.. Comment - `Link text <https://domain.invalid/>`_
+
+Collaborations Workshop 2020 (CW20) - Online 
+=============================================
+
+Case study authors: Rachael Ainsworth (`ORCID <http://orcid.org/0000-0003-2591-9462>`__) and Shoaib Sufi (`ORCID <https://orcid.org/0000-0001-6390-2616>`__)
+
+The Collaborations Workshop (CW) is the annual flagship event of the Software Sustainability Institute. It brings together researchers, developers, innovators, managers, funders, publishers, leaders, and educators to explore and share best practice.
```

Comment:
We might send people a link to the case study - it might be the first thing they see on EOG - so I think linking Software Sustainability Institute to www.software.ac.uk would be a good thing to do - even though it's a repeat of the link from the intro of the EOG